_posts/2020-02-05-keycloak-z-angularem-i-keycloakiem.md

```diff
@@ -0,0 +1,427 @@
+---
+layout:    post
+title:     "Keycloak - uwierzytelnianie i autoryzacja użytkownika w aplikacji Angular/Spring Boot"
+published: true
+date:      2020-02-05 08:00:00 +0100
+author:    mhoja
+tags:
+    - angular
+    - spring boot
+    - keycloak
+    - keycloak-spring-boot-2-starter
+    - keycloak-spring-boot-2-adapter
+    - keycloak-angular
+    - uwierzytelnianie użytkownika
+---
+
+Jak wykorzystać serwer Keycloak do logowania w naszej aplikacji?  
```

Comment:
w całym wpisze masz 22 wystąpienia "nasz", "nasze" itd. Co być powiedział na usunięcie tych wyrazów. Nie zmienia to znaczenia a moim zdaniem brzmi to lepiej, mniej "personalnie" :)

Reply:
Usunę niektóre, bo niektóre według mnie lepiej żeby zostały - sugerują że użyjemy klasy napisanej przez nas, a nie jakiejś np. z biblioteki keycloak-angular (są dwa serwisy, jeden napisany przez nas, drugi który udostępnia biblioteka).